features/user_views_artists.feature

```diff
@@ -1,7 +1,13 @@
 Feature: User views artists listing
+  Background:
+    Given the app has a few artists
 
   Scenario: A user views the listing of artists
-    Given the app has a few artists
     When I open the app
     And I want to see a listing of the artists
```

Comment:
These two steps

```
When I open the app
And I want to see a listing of the artists
```

are the same as

```
When I visit the artists page
```

We can move `When I visit the artists page` to the `Background` group. And maybe we can remove [this step definition](https://github.com/scooter-dangle/baltimore_public_art_commons/blob/add-artist-show-view/features/step_definitions/user_views_artists.rb#L8-L10)?